### ISSUE TITLE
Removing media profile causes torrent to have different infohash

### ISSUE THREAD

**Issue by CirnoT** [bhutanio/torrent-bencode#3]
Media profiles that are being removed are inside info dict - removing them will cause torrent to have different infohash

https://github.com/bhutanio/torrent-bencode/blob/63cd0aabd8b832f376a07f6bf25a8e52fa7b12ba/src/Bhutanio/BEncode/BEncode.php#L231

**Comment by abixalmon** [comment]
making it private already causes torrent to have different info_hash.

**Comment by CirnoT** [comment]
Correct, however trackers using it must be aware of it and make sure user redownloads torrent after upload if media profiles are detected. It may also be an issue if torrent was previously uploaded without removing media profile, set to private after upload and now is being downloaded by some other user (possibly years after original one was uploaded) - since the only way to set new announce_url is to execute make_private it will also remove media profile resulting in downloaded torrent file having different infohash than stored in tracker database.
